src/Synthesizer.ml

```diff
@@ -207,7 +210,7 @@ let solve_impl (config : Config.t) (task : task) (stats : stats) =
 
   let constants = Value.(
     List.dedup_and_sort ~compare
-       ( Value.[ Int 0 ; Int 1 ; Bool true ; Bool false ]
+       ( Value.[ Int 0 ; Int 1 ; Bool true ; Bool false ; Real 0.0 ; Real 1.0 ; Real 2.0 ]
        @ (List.map task.constants ~f:(function Int x -> Int (abs x) | x -> x))))
```

Comment:
Is there a special reason to include 2.0?

Reply:
When I was adding constants, Saswat recommended to include 2.0 to be able to synthesize fractions faster.

Reply:
Yeah with just 0 and 1, division is kind of useless. You'd at least have to synthesize 1+1 to make useful divisions.